src/structurizr/workspace.py

```diff
@@ -60,46 +61,49 @@ class WorkspaceIO(BaseModel):
 
     """
 
-    id: int = Field(..., description="The workspace ID.")
+    id: int = Field(default=None, description="The workspace ID.")
```

Comment:
Isn't it required?

Reply:
Not sure, the id of the workspace is used only for submission, which is also being populated by the client. Opened a ticket: #21